src/debuginfo.cpp

```diff
@@ -287,7 +287,7 @@ void JITDebugInfoRegistry::registerJITObject(const object::ObjectFile &Object,
     uint8_t *catchjmp = NULL;
     for (const object::SymbolRef &sym_iter : Object.symbols()) {
         StringRef sName = cantFail(sym_iter.getName());
-        if (sName.equals("__UnwindData") || sName.equals("__catchjmp")) {
+        if (sName == "__UnwindData" || sName == "__catchjmp") {
```

Comment:
In which version of llvm was this operator introduced? We currently still need to support llvm 15 on master (llvm version of latest stable series), or llvm 16 by the time Julia v1.11 will be released (hopefully soon enough)

Reply:
I believe it was there 15 years ago. I can confirm when I'm next at a computer.

Reply:
It was introduced with StringRef, https://github.com/llvm/llvm-project/commit/25f9fc58511807f712d7789a09157da2986f52d1, so llvm 2.